Gems/AWSGameLift/Scripts/gamelift_build_package_linux.py

```diff
@@ -0,0 +1,169 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Python script to create the package necessary for GameLift
+# To do this by hand please see the steps provided @ https://www.o3de.org/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-linux/
+
+import os 
+import shutil
+import re
+import argparse
+import itertools
+import threading
+import time
+import sys
+import stat
+
+def main():        
+    script_description = """Create a Linux dedicated server package for GameLift\n
+    Prerequisites:
+    - You have built your project with the AWS GameLift Gem enabled.
+    - You have built the Profile version of your project's server launcher.
+    - You have run the Asset Processor and compiled all of the project's assets.
+    https://www.o3de.org/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-linux/"""
+    parser = argparse.ArgumentParser(description = script_description, formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('--package_dir', help='where to create the linux package', required=True)
+    parser.add_argument('--profile_build_dir', help="your game's build profile folder", required=True)
+    parser.add_argument('--asset_cache_dir', help="your game's asset cache folder", required=True)
+
+    parser.parse_args()
+
+    package_dir = parser.parse_args().package_dir
+    build_bin_folder = parser.parse_args().profile_build_dir
+    project_cache_folder = parser.parse_args().asset_cache_dir
+
+    if not project_cache_folder.endswith("Cache") and not project_cache_folder.endswith("Cache/"):
+        print("Package cancelled. Invalid asset_cache_dir")
```

Comment:
include invalid path in error message. Consider using python logger rather than just raw print